### DISCUSSION TITLE
Welcome to snowpack-svelte-ts-tw Discussions!

### DISCUSSION THREAD

**Discussion by GarrettCannon** [GarrettCannon/snowpack-svelte-ts-tw#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stevesob** [comment]
Hello, I'm Steve, the CEO at OLIS.COM (www.uscourts.com), a leader in Court CMS Systems.

I'm looking for a couple of knowledgeable developers to consult with my long time very experienced .Net developers. We are transitioning from WebForms to Svelete for our client side web apps.

If you would like to earn some extra income and have time to answer questions and guide my developers to the JavaScript ecosystem, please get in touch. Contact me at steve@olis.com
